mne/epochs.py

```diff
@@ -149,7 +150,7 @@ def __init__(self, info, event_id, tmin, tmax, baseline=(None, 0),
                        'result in a sampling frequency of %g Hz, which can '
                        'cause aliasing artifacts.'
                        % (lowpass, decim, new_sfreq))  # 50% over nyquist limit
-                warnings.warn(msg)
+            warnings.warn(msg)
```

Comment:
Fixed a little bug here. If `lowpass == None` a warning should have been generated, but was not. This was never covered by the tests.